Sources/AppcuesKit/Presentation/ExperienceRendering/ExperienceTrigger.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ExperienceTrigger.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 12/8/22.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import Foundation
+
+internal enum ExperienceTrigger {
+    case qualification(reason: String?)
```

Comment:
No reason to not use the enum - nice improvement!

This code actually dates back to some previous discussions around tracking analytics for how flows are triggered for user journey mapping. This associated value is not used at all currently - so I'm not opposed to just taking it out under the rules of YAGNI.

However, it seemed fairly likely that similar future needs for this type of data would come up, so that's why I kept it along for the ride.